### PR TITLE
Fix Model Attribute handler bug and improve perf

### PR DESCRIPTION
**Pull request by vicancy**

### CODE REVIEW

File: test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs [hunk 1]
```diff
@@ -222,14 +282,30 @@ private static HandleModelAttributesContext GetDefaultContext()
...
             public string Content { get; set; }
 
             public string Content2 { get; set; }
 
-            public List<string> Content3 { get; set; }
+            [MarkdownContent]
+            public string ReadonlyContent { get; } = "*content";
+
+            [UniqueIdentityReference]
+            [UniqueIdentityReferenceIgnore]
```
Comment (superyyrrzz) [inline test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs]: why have both?

**Comment by vwxyzh** [comment]
:shipit:

**Comment by hellosnow** [comment]
:shipit:

File: test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs (hunk 1 above)
Comment (vicancy) [inline test/Microsoft.DocAsCode.Build.Common.Tests/ModelAttributeHandlerTest.cs]: To test that UniqueIdentityReferenceIgnore takes effect over UniqueIdentityReference

**Pull request merged**